src/cli.ts

```diff
@@ -189,11 +187,12 @@ const cli = async (): Promise<void> => {
 
     const sao = new SAO({
         generator,
-        outDir: finalProjectDir,
+        outDir: process.cwd(),
```

Comment:
Can we try this with `npm run build:global` (with global install) and running the executable from the globally installed one? 🤔